### PR TITLE
Members scenarios

### PR DESCRIPTION
**Pull request by nadimdiazramzi**

### CODE REVIEW

File: features/members/delete_member.feature [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    |key |value    |
+    |name|[NDR]Board Test02 | 
+And I add a new member to "/boards/{id}/members"
+    |key |value    |
+    |fullName|Pepito Perez |
+    |email|pepito@gmail.com|
+    |type|normal|
+When I delete a member using "/boards/{id}/members/{idMember}"
+Then the status code should be "401"
+And I send a GET request to "/boards/{id}/members"
```
Comment (jmaldonado-rojas) [inline features/members/delete_member.feature:15]: And => cambiar a When

File: features/members/delete_member.feature [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    |key |value    |
+    |name|[NDR]Board Test02 | 
+And I add a new member to "/boards/{id}/members"
+    |key |value    |
+    |fullName|Pepito Perez |
+    |email|pepito@gmail.com|
+    |type|normal|
+When I delete a member using "/boards/invalid/members/{idMember}"
+Then the status code should be "400"
+And I send a GET request to "/boards/{id}/members"
```
Comment (jmaldonado-rojas) [inline features/members/delete_member.feature:30]: And => Reemplazar por When

File: steps/members_steps.py [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    endpoint = endpoint.replace('{id}',idRecord)
+    context.response = request_util.put_request(context.base_url + endpoint, query_params)
+
+    #Save member Id to be used later
+    data = context.response.json()
+    idRecord = data['id']
+    context.recordId = idRecord
+
+@when(u'I send a GET request to "{endpoint}"')
+@then(u'I send a GET request to "{endpoint}"')
```
Comment (jmaldonado-rojas) [inline steps/members_steps.py:41]: @then(u'I send a GET request to "{endpoint}"'), esta mejor sin esto

File: steps/members_steps.py [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+def step_impl(context,endpoint):
+    query_params = ({"key": context.key}, {"token": context.token})
+
+    endpoint = endpoint.replace('{id}',context.boardId)
+    context.getResponse = request_util.get_request(context.base_url + endpoint, query_params)
+    context.response = context.getResponse
+
+@when(u'I delete a member using "{endpoint}"')
```
Comment (CristelhM) [inline steps/members_steps.py:49]: you may use this implementation already defined:
@step(u'I send a {method} request to "{endpoint}"')

File: steps/members_steps.py [hunk 5]
```diff
@@ -0,0 +1,78 @@
+from utils import request_util
+import json
+from compare import expect
+import collections
+
+@given(u'I send a POST request to "{endpoint}"')
```
Comment (CristelhM) [inline steps/members_steps.py:6]: you may use this: @step(u'I send a {method} request to "{endpoint}"')

**Pull request merged**